src/initialization/FGInitialCondition.cpp

```diff
@@ -590,6 +590,11 @@ void FGInitialCondition::SetWindDownKtsIC(double wD)
   calcAeroAngles(_vt_NED);
 }
 
+void FGInitialCondition::SetWindMagFpsIC(double mag)
+{
+  SetWindMagKtsIC(mag*fpstokts);
+}
+
```

Comment:
Since the code is trivial, I'd rather move that definition into the header so that the C++ compiler can inline it when possible (see my other comment below):
```suggestion
```

Reply:
Will do, although C++ compilers/linkers have been able to inline methods in .cpp files for many years :wink:

Reply:
I'll add an additional test to the unit test.

Reply:
> Will do, although C++ compilers/linkers have been able to inline methods in .cpp files for many years 😉

OK I'm lagging many years behind :smile: but I've always had this idea that when the body of a method is given in the header then compilers would intepret that as an `inline` requirement (that they can happily ignore by the way). Anyway that doesn't hurt either way but having a one-liner body in the header saves the trouble of opening the `.cpp` file :wink: